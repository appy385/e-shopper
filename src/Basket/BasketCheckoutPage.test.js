import React from 'react';
import { render } from '@testing-library/react';
import BaksetCheckout from './BasketCheckoutPage';

describe(BaksetCheckout.name, () => {
  let mockBasket;
  let mockAddOrder;
  beforeEach(() => {
    mockBasket = {
      'Fruits & Vegatables': [{
        id: 1,
        src: 'https://images.pexels.com/photos/61127/pexels-photo-61127.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500',
        seller: 'Fresho',
        name: 'Banana-Robusta',
        price: 40,
        count: 2,
        category: 'Fruits & Vegatables',
      }],
    };
    mockAddOrder = jest.fn();
  });
  test('shoudl match snapshot', () => {
    const { container } = render(<BaksetCheckout basket={mockBasket} addOrder={mockAddOrder} />);
    expect(container).toMatchSnapshot();
  });
});
