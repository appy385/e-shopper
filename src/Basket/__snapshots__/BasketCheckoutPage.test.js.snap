// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasketCheckoutPage shoudl match snapshot 1`] = `
<div>
  <div>
    <h1>
      Please Fill in Details!!
    </h1>
    <form
      action="#"
      class="checkout-form"
    >
      <div>
        <label
          for="firstName"
        >
          First Name:
        </label>
        <input
          id="firstName"
          name="firstName"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          for="lastName"
        >
          Last Name:
        </label>
        <input
          id="lastName"
          name="lastName"
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          for="email"
        >
          Email Address:
        </label>
        <input
          id="email"
          name="email"
          type="email"
          value=""
        />
      </div>
      <button
        id="submit"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
